docs/rules/sort-comp.md

```diff
@@ -92,7 +99,7 @@ The default configuration is:
 * `type-annotations`. This group is not specified by default, but can be used to enforce flow annotations positioning.
 * `getters` This group is not specified by default, but can be used to enforce class getters positioning.
 * `setters` This group is not specified by default, but can be used to enforce class setters positioning.
-* `instance-variables` This group is not specified by default, but can be used to enforce all other instance variables positioning.
+* `instance-variables` This group can be used to enforce all other instance variables' positioning.
```

Comment:
specifying this by default makes this a breaking change; please revert that piece.